test.js

```diff
@@ -130,16 +134,82 @@ test('stripFinalNewline: false', async t => {
 	t.is(stdout, 'foo\n');
 });
 
-test('stripFinalNewline on failure', async t => {
-	const {stderr} = await t.throwsAsync(execa('noop-throw', ['foo'], {stripFinalNewline: true}));
-	t.is(stderr, 'foo');
+test('execa() with .kill after it with SIGKILL should kill cleanly', async t => {
+	const subprocess = execa('node', ['fixtures/no-killable'], {
+		stdio: ['ipc']
+	});
+
+	await waitForProcessLaunch(subprocess);
+
+	subprocess.kill('SIGKILL');
+
+	const {signal} = await t.throwsAsync(subprocess);
+	t.is(signal, 'SIGKILL');
 });
 
+// Windows doesn't support sending signals. No re-emulates them down to SIGKILL
+if (process.platform !== 'win32') {
```

Comment:
Instead of repeating this comment and condition several times, could you please regroup them?

```js
// Windows doesn't support sending signals. No re-emulates them down to SIGKILL
if (process.platform !== 'win32') {
  test(...)

  test(...)

  test(...)
}
```

Reply:
Yes, of course. Weird that I didn't do it like that right away: 00ad35962d5b2960fd3364737ee647eb1035211d